code/modules/client/preferences/entries/player/chat.dm

```diff
@@ -80,3 +80,14 @@
 	if (!..())
 		return FALSE
 	return is_admin(preferences.parent)
+
+/datum/preference/toggle/examine_messages
+	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
+	db_key = "examine_messages"
+	preference_type = PREFERENCE_PLAYER
+
+/datum/preference/toggle/whole_word_examine_links
+	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
+	db_key = "whole_word_examine_links"
+	preference_type = PREFERENCE_PLAYER
+	default_value = FALSE
```

Comment:
```suggestion
	default_value = TRUE
```

This should default to TRUE as the links provide a less confusing experience and give some syntactic highlighting which make it easier to read. These preferences will never realistically be changed, so we'd better use the option that provides a better experience for the user.

Reply:
> that provides a better experience for the user

Entirely subjective, but I acquiesce